block-everything.txt

```diff
@@ -471,7 +476,13 @@ smashingmagazine.com##.c-promo-box--ad
 space.com##.post-gallery-item-ad
 spectator.us##.article_set_wrap:style(position: static !important;)
 spectator.us##.otherheader:style(position: absolute !important;)
+stackexchange.com##.tex2jax_ignore
+stackexchange.com##.s-anchors
```

Comment:
This is too broad. The Share Edit Follow Flag bar blocks useful links such as Edit and Flag.

Reply:
I hadn't noticed that .s-anchors albo blocks that bar. I think it should be .s-sidebarwidget__yellow, but if you don't mind Hot Network Questions, then I'll also leave Featured on Meta.